syndeo-template/src/main/java/com/google/cloud/syndeo/GenerateConfiguration.java

```diff
@@ -177,4 +193,17 @@ static com.google.cloud.datapipelines.v1.syndeo.TransformDescription providerToC
             String.format("Unable to convert Beam type %s to Datapipelines type.", beamFieldType));
     }
   }
+
+  static String urnToTransformLabel(String urn) {
```

Comment:
Created an additional HashMap that maps supported URNs to the Transform label. The format of the label would simply be the IO name, since the Read/Write part is inherently determined whether the IO is being used as a source or sink